internal/capability/operator_cleanup.go

```diff
@@ -21,12 +22,27 @@ func operatorCleanup(ctx context.Context, opts ...auditOption) auditCleanupFn {
 
 	return func(ctx context.Context) error {
 		// delete subscription
+		SubscriptionList := &operatorv1alpha1.SubscriptionList{}
```

Comment:
Is this meant to be exported?

Reply:
You mean the upper case SubscriptionList right? @yashoza19 when you can just check if it's really needed.

Reply:
Yes correct, it looks like a function local variable that is being exported.